pulsar-flink-connector/src/main/java/org/apache/flink/streaming/connectors/pulsar/table/PulsarTableOptions.java

```diff
@@ -170,7 +170,13 @@ private PulsarTableOptions() {
             .key("scan.startup.sub-name")
             .stringType()
             .noDefaultValue()
-            .withDescription("Optional sub-name used in case of \"specific-offsets\" startup mode");
+            .withDescription("Optional sub-name used in case of \"external-subscription\" startup mode");
+
+    public static final ConfigOption<String> SCAN_STARTUP_SUB_START_OFFSET = ConfigOptions
+            .key("scan.startup.sub-startOffset")
```

Comment:
rename config name to `scan.startup.sub-start-offset`, the key configured in the with block of flink sql needs to be lowercase.